### ISSUE TITLE
Training requirements

### ISSUE THREAD

**Issue by terrarier2111** [arimousa/DDAD#45]
Which GPU was used to train this model and how much VRAM is approximately required to train this model?
Is VRAM usage during training/inference is related to sample size?

**Comment by arimousa** [comment]
We used  GeForce RTX 3090, 24 Ram. The VRAM determines the batch size we can fit to the memory. We used 2 GPUs to train the model.

**Comment by terrarier2111** [comment]
Alright thanks for the quick response, do you think a single GPU with 24GB memory would suffice?

**Comment by arimousa** [comment]
You can use gradient accumulation. Lower batch size is usually sufficient but you may get better results if you have a reasonable batch size.
